amethyst_core/src/transform/systems.rs

```diff
@@ -56,9 +78,26 @@ impl<'a> System<'a> for TransformSystem {
                 .expect("unreachable");
         }
 
+        self.isom_modified.clear();
         self.local_modified.clear();
         self.global_modified.clear();
 
+        // Gather the inserted and mutated IsometricTransforms in one BitSet.
+        isoms.populate_inserted(
+            self.inserted_isom_id.as_mut().unwrap(),
+            &mut self.isom_modified,
+        );
+        isoms.populate_modified(
+            self.modified_isom_id.as_mut().unwrap(),
+            &mut self.isom_modified,
+        );
+
+        // Apply IsometricTransforms to their associated Transform.
+        for (_, isom, mut local) in (&self.isom_modified, &isoms, &mut locals).join() {
+            let v = isom.local();
+            local.translation = v.extend(-v.y + isom.order_priority);
```

Comment:
What's the motivation behind this one?

Reply:
Reading the example code: so extending the vec2 to a vec3, where z is encodes the drawing order (somehow).

Reply:
That's basically Z ordering, that's how the sprite batch does it.